src/mothd.go

```diff
@@ -41,6 +41,12 @@ func main() {
 		"/theme",
 		"Path to static theme resources (HTML, images, css, ...)",
 	)
+	flag.StringVar(
+		&ctx.Progression,
+		"progression",
+		"all",
+		"The type of challenge progression (team | all)",
+	)
```

Comment:
You mean making the config a file in the state directory?  I think it is an appropriate place, and with the current library we use we would just need to specify that directory.

Reply:
I mean something like

```go
teamProgression := false
if _, err := os.Stat(ctx.StatePath("disabled")); err == nil {
	teamProgression = true
}
```

Reply:
Not married to making it a state file, just wanted to bounce it off you.

Reply:
Ooh I see what you mean.  We could do that, most definitely.  There is something nice about storing it in the per-competition state rather than in a config.  Maybe we should just have a config in state, which has per-competition configuration?